types/openlayers/index.d.ts

```diff
@@ -7918,7 +7919,7 @@ export namespace render {
              * @param style The rendering style.
              * @api
              */
-            setStyle(style: ol.style.Style): void;
+            setStyle(style: ol.style.Style | null): void;
```

Comment:
According to the [source code](https://github.com/openlayers/openlayers/blob/07cb678271d725504762b9fa12f59a99c660dc0c/src/ol/render/canvas/Immediate.js#L422), this is invalid. If you pass `null` to this function, it will fail.

Reply:
You are right, this function should have been kept unchanged. I am reverting the change.